Makefile

```diff
@@ -40,6 +40,11 @@ test-php-integration: run-ocis-with-keycloak
 	composer install
 	$(call run-with-cleanup, $(PHPUNIT) --configuration ./phpunit.xml --testsuite integration, $(MAKE) docker-clean)
 
+.PHONY: test-single-php-integration
+test-single-php-integration:             ## Run single php integration test
+	composer install
+	$(PHPUNIT) --configuration ./phpunit.xml --testsuite integration --filter $(filter-out $@,$(MAKECMDGOALS))
```

Comment:
can we put that into the normal `test-php-integration` goal?
check if there are any extra arguments, and if yes add `--filter` and if not omit it